### PR TITLE
Update mergify config to require review by Randy unless he authored

### PR DESCRIPTION
**Pull request by puiterwijk**
Signed-off-by: Patrick Uiterwijk <patrick@puiterwijk.org>

### CODE REVIEW

File: .mergify.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ pull_request_rules:
       strict: true
   conditions:
   - label!=no-mergify
-  - '#approved-reviews-by>=1'
+  - approved-reviews-by=bowlofeggs
```
Comment (cverna) [inline .mergify.yml]: What happens when Randy takes some vacations ? Can we still merge manually ?

**Comment by bowlofeggs** [comment]
On Thu, 2018-11-15 at 07:39 -0800, Clement Verna wrote:
> What happens when Randy takes some vacations? Can we still merge
> manually?

Yes, manual merging is possible for an administrator. In general, I
prefer PRs not to get merged without my review unless they are critical
(security or severe bug).

**Comment by bowlofeggs** [comment]
I'm going to rebase this on top of https://github.com/fedora-infra/bodhi/commit/41c794e4a6f452b92e8146af126d277055fe34b6.

**Comment by bowlofeggs** [comment]
I am also removing f27 from the new rules since that was recently dropped from testing.

**Pull request merged**